### PR TITLE
Layer1 code style for PR

### PR DESCRIPTION
**Pull request by nsmith-**
For cms-sw/#13767

### CODE REVIEW

File: L1Trigger/L1TCaloLayer1/src/UCTTower.hh [hunk 1]
```diff
@@ -5,24 +5,26 @@
...
-#define ecalFlagMask 0x00008000
-#define stg2BitsMask 0x0000FFFF
-#define miscBitsMask 0x0000F000
-#define miscShift 12
-
-#define ecalBitsMask 0x00FF0000
-#define ecalShift 16
-#define hcalBitsMask 0xFF000000
-#define hcalShift 24
+namespace {
```
Comment (fwyzard) [inline L1Trigger/L1TCaloLayer1/src/UCTTower.hh]: I'm not sure using an anonymous namespace is the right choice,  but maybe @davidlange6 can comment.

My impression is that it does protect from linker clashes (the symbols should visible only within the file that defines them), but not from clashes with other symbols with the same name.

Comment (nsmith-) [inline L1Trigger/L1TCaloLayer1/src/UCTTower.hh]: Here's a list of over 1000 `#define`s in header files in CMSSW
http://ncsmith.web.cern.ch/ncsmith/cmssw_header_defines.txt
I cannot fathom why you are so insistent on having these fixed when they are in private header files in private directories, with no intention for anyone to use them just as much as any of the header files in the list I linked, especially when you so often re-iterate the time sensitivity of this PR.

Comment (fwyzard) [inline L1Trigger/L1TCaloLayer1/src/UCTTower.hh]: Sorry, I'm not sure if this is supposed to be a tentative of a provocative statement, or if I should really answer ?

Comment (fwyzard) [inline L1Trigger/L1TCaloLayer1/src/UCTTower.hh]: @nsmith- , could you please change the anonymous namespace

``` c++
namespace {
```

to something like

``` c++
namespace l1tcalo {
```

or

``` c++
namespace  l1t {
    namespace calo {
```

and add the corresponding `using` statements to the source files, i.e.

``` c++
using namespace l1tcalo;
```

or

``` c++
using namespace l1t::calo;
```

?

Thank you,
.Andrea

**Pull request merged**